tests/unit/browser/test_pdfjs.py

```diff
@@ -182,7 +182,7 @@ def test_read_from_system_oserror(tmpdir, caplog, unreadable_file):
     assert message.startswith('OSError while reading PDF.js file:')
 
 
-@pytest.mark.parametrize('available', [True, False])
+@pytest.mark.parametrize('available', ['pdf.js', None])
 def test_is_available(available, mocker):
```

Comment:
`is_available` is patched in various other locations in this file as well, which probably should be changed accordingly too.

Reply:
as the function retains it original signature now, the patching has been made superfluous.